DS13/code/modules/overmap/ai.dm

```diff
@@ -145,7 +145,17 @@
 		nav_target = target
 		if(cloaked) //No firing while cloaked, please!
 			return
+		
+		var/shouldSwitchWeapons = rand(0, 100)
+		if(shouldSwitchWeapons > 66) // 1 in 3 chance of switching weapons every 2 seconds
+			if(fire_mode == "phaser")
+				if(photons > 0)
+					fire_mode = "torpedo"
```

Comment:
Is this the right name? can't remember if it's that or photon

Reply:
The attack code checks if it's a phaser, else it assumes torpedo.